Makefile

```diff
@@ -163,3 +163,13 @@ $(AUTOGENERATED_BLOCKLIST_CLIENT_CLIENT): $(BLOCKLIST_OPENAPI_SPEC)
 # Generate the OpenAPI spec for Blocklist Client
 $(BLOCKLIST_OPENAPI_SPEC): $(INSTALL_TARGET) $(filter-out $(BLOCKLIST_OPENAPI_SPEC), $(wildcard $(subst dir, $(BLOCKLIST_OPENAPI_PATH), $(THREE_DIRS_DEEP))))
 	cd ./.generated-sources/blocklist-openapi-gen && cargo build
+
+# Local Docker Compose Devenv
+
+devenv-up: $(INSTALL_TARGET) $(EMILY_LAMBDA_BINARY) $(EMILY_CDK_TEMPLATE)
```

Comment:
Interesting - I'm honestly not sure. It's possible that adding `EMILY_LAMBDA_BINARY` here causes more pain than it helps if it requires a more customized command (although I'm guessing the makefile supports that kind of thing).

Reply:
Oh interesting, I just tried to re-build it and hit the same issue. I had to comment out the `ifneq` block that defines `_LAMBDA_FLAGS` for it to work, too. @AshtonStephens should we just scrap these three lines?

https://github.com/stacks-network/sbtc/blob/3a64a95170f227169fc628b42448a92789d20132/Makefile#L135-L137

Reply:
And, yeah, I see that error response too. Looking at the logs, it's from this:

`Invalid lambda response received: Lambda response must be valid json`

I'm not sure how to follow up on this error - Ashton lmk if you have thoughts here too

Reply:
I can debug this separately if necessary

Reply:
That flag is there because `cargo lambda` compiles with `x86` architecture by default regardless of the local machine. It does this because the default lambda container for non-standard binaries (rust counts as one of those) is an `x86` machine. To run it locally I had to set that flag specifically.